release/karaf/itests/switchyard/pom.xml

```diff
@@ -377,7 +377,7 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-deploy-plugin</artifactId>
                 <configuration>
-                    <skip>true</skip>
+                    <skip>false</skip>
```

Comment:
yep, that looks better.
Since the same PR have already been merged to 1.0.x -1.3.x branch. They had used <skip>false</skip>
To remove the skip configuration probably would make the code a bit inconsistency. 

Maybe we could remove the skip config next time when we need to update the jb-ip-bom version for all other branch.